build-info-extractor-docker/src/test/java/org/jfrog/build/extractor/docker/extractor/DockerExtractorTest.java

```diff
@@ -31,44 +27,33 @@
 import java.nio.file.Paths;
 import java.nio.file.attribute.PosixFilePermission;
 import java.nio.file.attribute.PosixFilePermissions;
-import java.util.Collections;
-import java.util.List;
+import java.util.*;
 
-import static org.testng.Assert.assertEquals;
-import static org.testng.Assert.assertFalse;
-import static org.testng.Assert.assertNotNull;
-import static org.testng.Assert.assertTrue;
+import static org.testng.Assert.*;
 
 @Test
 public class DockerExtractorTest extends IntegrationTestsBase {
     private static final Path PROJECTS_ROOT = Paths.get(".").toAbsolutePath().normalize().resolve(Paths.get("src", "test", "resources", "org", "jfrog", "build", "extractor", "docker"));
     private static final String PROJECT_PATH = PROJECTS_ROOT.toAbsolutePath().toString();
+    private static final String DOCKER_LOCAL_REPO = "build-info-tests-docker-local";
+    private static final String DOCKER_REMOTE_REPO = "build-info-tests-docker-remote";
+    private static final String DOCKER_VIRTUAL_REPO = "build-info-tests-docker-virtual";
     private static final String SHORT_IMAGE_NAME = "jfrog_artifactory_buildinfo_tests";
     private static final String SHORT_IMAGE_TAG_LOCAL = "2";
     private static final String SHORT_IMAGE_TAG_VIRTUAL = "3";
     private static final String EXPECTED_REMOTE_PATH_KANIKO = "hello-world/latest";
-
-    private static final String LOCAL_DOMAIN = "BITESTS_ARTIFACTORY_DOCKER_LOCAL_DOMAIN";
-    private static final String REMOTE_DOMAIN = "BITESTS_ARTIFACTORY_DOCKER_REMOTE_DOMAIN";
-    private static final String VIRTUAL_DOMAIN = "BITESTS_ARTIFACTORY_DOCKER_VIRTUAL_DOMAIN";
-    private static final String DOCKER_LOCAL_REPO = "BITESTS_ARTIFACTORY_DOCKER_LOCAL_REPO";
-    private static final String DOCKER_REMOTE_REPO = "BITESTS_ARTIFACTORY_DOCKER_REMOTE_REPO";
-    private static final String DOCKER_VIRTUAL_REPO = "BITESTS_ARTIFACTORY_DOCKER_VIRTUAL_REPO";
-    private static final String DOCKER_HOST = "BITESTS_ARTIFACTORY_DOCKER_HOST";
```

Comment:
Doesn't removing the support for host means running on macos is not longer supported?